Pod/Classes/MenuView.swift

```diff
@@ -268,11 +268,22 @@ public class MenuView: UIScrollView {
         return menuItemViews[currentPage].frame.midX - UIScreen.mainScreen().bounds.width / 2
     }
     
+    
     private func contentOffsetXForCurrentPage() -> CGFloat {
-        guard menuItemViews.count > options.minumumSupportedViewCount else { return 0.0 }
-        
-        let ratio = CGFloat(currentPage) / CGFloat(menuItemViews.count - 1)
-        return (contentSize.width - frame.width) * ratio
+        var offset: CGFloat = 0
+        if self.frame.width > self.contentSize.width {
+            return offset
+        }
+        if (currentPage == 0 && currentPage != 0) || (currentPage > 0 && currentPage != 0) {
```

Comment:
This is not a bug at this moment. If you have a few items, why don't you use SegmentedControl mode?
Anyway, I don't have any plan to change this behaviour for now.